scripts/lint/markdown-heuristic-lint.py

```
@@ -0,0 +1,320 @@
+_GLOB_PATTERN = re.compile(
+    r'(?<!\*\*/)(?<!\*/)(?<!\.\*\*)(?<!\./\*\*)(?<!\w)'
+    r'(\*\.[a-zA-Z0-9_]+)'
+)
+
+# ---------------------------------------------------------------------------
+# COUNT001 configuration
+# ---------------------------------------------------------------------------
+# Window (in lines) after the narrative phrase to search for the list.
+COUNT001_WINDOW = 30
+
+# Written-out numerals 0–20.
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)